config/schema.sql

```diff
@@ -103,6 +103,9 @@ CREATE TABLE oauth_data (
     code text NOT NULL
 );
 
+CREATE UNIQUE INDEX on oauth_data (code);
+ALTER TABLE oauth_data ADD CONSTRAINT unique_code UNIQUE (code);
```

Comment:
It's not necessary to have both, but momentarily just added both lines since I wasn't sure what the preference between the two would be. Might be better just to keep the INDEX.